app/routes/_layout+/assets.$assetId.overview.tsx

```diff
@@ -268,6 +268,15 @@ export default function AssetOverview() {
                 </div>
               </li>
 
+              {asset?.propertyId ? (
```

Comment:
Indentation is broken. Make sure you use prettier so your formatting is consistent with our codebase. Also when you run the commit message, our post-commit hooks should also fix the formatting so you can also do it this way.